crt/README.md

```diff
@@ -65,7 +65,7 @@ The top of the stack is structured as so:
 * unspecified data
 
 For simplicity, we simply pass a pointer to the stack to the C entry in
-the runtime and dissasemble the stack there. The entry we use is as
```

Comment:
The stack isn't really being assembled at all in [`__sel4_start_c`](https://github.com/SEL4PROJ/sel4runtime/blob/4e4705c7503ea82ed8611fb4f597e66706b57309/src/crt1.c#L30). 'traversed' might be a better term to use.

Reply:
'hand-crafted' is what I Ihink I'll use.

Reply:
But nothing is being created here. This is referring to the fact that the code is just finding the addresses to content already on the stack